pkg/controller/controller_utils_test.go

```diff
@@ -585,8 +685,9 @@ func TestSortingActivePodsWithRanks(t *testing.T) {
 		{lesser: podWithRank{highPodDeletionCost, 2}, greater: podWithRank{lowPodDeletionCost, 1}, disablePodDeletioncost: true},
 		{lesser: podWithRank{ready2Hours, 1}, greater: podWithRank{ready5Hours, 1}},
 	}
-	for i, test := range inequalityTests {
-		t.Run(fmt.Sprintf("test%d", i), func(t *testing.T) {
+
+	t.Run("Inequality tests", func(t *testing.T) {
```

Comment:
I'm not sure if this swap of the order for the `for` and `t.Run` is expected. I think if the `t.Run` is inside it can give more contextual information about the test case name. I see that currently we don't provide better names, but this is something we could actually improve, potentially in a follow up.

Reply:
Right now, in case the test case passes you'll get a single "PASS" message, and in case of a failure, you also get a single "FAIL" message with an extra context in the failure messages. If the `for` and `t.Run` were swapped, you'll get as many messages as inequality tests, all with the same name (which, IMO adds no meaningful information).

This is not great at all, but I think the swap would make more sense if each test had a proper name. For this first refactor iteration, my intention was to introduce the test tables and modify the less code possible to keep the coverage, so I propose to do this in a follow-up along with a proper code reorganization.

Reply:
Yes, these test cases should be named. Until then, I prefer to leave as it was - name the tests based on their indexes as `fmt.Sprintf("test%d", i)`.

Reply:
ok! No problem, I'll do that. If that's ok for you, I'll name each test as "Inequality tests %d" since think it gives more context